modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+In embedded mode, the avatar is supplied by the `tinycomments_author_avatar` option:
+
+[source,js]
+----
+    plugins: [
+        "tinycomments",
+    ],
+	tinycomments_mode: "embedded",
+	tinycomments_author: "testuser",
+	tinycomments_author_name: "Test User",
+	tinycomments_author_avatar: "https://example.com/image.ext",
+----
+
+In callback mode, the avatar is supplied by the `avatar` option of `setConverstation()`:
+
+[source,js]
+----
+const avatarUrl = "https://example.com/image.ext";
+
+/*
+
+much non-avatar-related configuration here
+
+*/
+
+      setConversation(conversationUid, [
+        {
+          author,
+		  avatar: avatarUrl,
+          createdAt,
+          modifiedAt: createdAt,
+          content,
+          uid: conversationUid /* first comment has same uid as conversation */,
+        },
+      ]);
+----
+
+In both modes the avatar:
+
+* must be 36px by 36px.
+
+* can be any filetype able to be wrapped in an `<img>` element.
+
+* will present as the browser’s default ‘image not found’ icon if the avatar resource is not available at the nominated URL.
+
+NOTE: Enablng this feature depends on changes made to the Core {productname} editor. As a consequence, showing avatars next to the user-name in comments requires both {productname} 6.1 or later and version 3.1.0 or later of the xref:comments.adoc[Comments Premium Plugin].
+
+
+==== Show comments sidebar on editor initialization
+
+It is now possible to initialize a {productname} editor with a comment sidebar automatically visible and available but with focus still in the main editor.
+
+To add a comment sidebar add `sidebar_show: 'showcomments'` to `tinymce.init`:
+
+[source,js]
+----
+tinymce.init({
+    selector: "textarea",
+	sidebar_show: 'showcomments',
+});
+----
+
+For a customised comments sidebar, setup the sidebar in the `setup` section of `tinymce.init`:
+
+[source,js]
+----
+	setup: (editor) => {
+	  editor.ui.registry.addSidebar('showcomments', {
+		tooltip: 'showcomments',
+		icon: 'comment',
+		onSetup: (api) => {
+		  console.log('Render panel', api.element());
+		},
+		onShow: (api) => {
+		  console.log('Show panel', api.element());
+		},
+		onHide: (api) => {
+		  console.log('Hide panel', api.element());
+		}
+	  });
+	}
+----
+
+NOTE: The comment sidebar presents with no animation or other effect. It is just there, along with any other initialised {productname} editor components. Also, as noted above, the sidebar does not take focus, which remains with the main editor.
+
+
+==== Support annotating certain block elements directly when using the editor annotator API
+
+Several new block elements can now have comments attached to them, including:
+
+* Tables of contents
+* Images
+* Images with captions
+* Code samples
+* media in iframes
+* embedded media
+* embedded pages
+
+When comments are showing, a 3px wide yellow (hex color #FFE89D) rectangle presents around any element with a comment attached.
```

Comment:
There are different shades of yellow highlight for selected and unselected commented elements

Reply:
It can also change based on the skin/content css, so I think we'd just be better to drop the hex color code.

Reply:
Edited out the color code. Added slighly more generic info about the defaults and their changeability. To wit:

`When comments are showing, a 3px wide colored rectangle presents around any element with a comment attached. By default this rectangle presents in two shades of yellow: one for commented elements which are selected and one for commented elements which are not selected. Using a Skin can, however, change this default.`